omnibus/cookbooks/omnibus-supermarket/.kitchen.yml

```diff
@@ -12,7 +12,6 @@ provisioner:
     ssl_verify_mode: verify_peer
 
 platforms:
-  - name: ubuntu-12.04
   - name: ubuntu-14.04
   - name: ubuntu-16.04
   - name: centos-5.11
```

Comment:
We still doing RHEL 5?

Reply:
The answer was "yes" last time I asked, but maybe it's time to ask again. RHEL5 isn't on the [Platforms and Supported Versions list](https://docs.chef.io/platforms.html) any more, so maybe not.

Reply:
We killed it Jan 1st